policy.md

```diff
@@ -1,181 +1,122 @@
 # Common CCADB Policy #
```

Comment:
Is "Common CCADB" redundant? Should the header read "CCADB Policy"?

Reply:
It's meant to indicate that the policy applies to all root store members of the CCADB.
Can remove the extra "Common" if you think that the scope of the policy sufficiently describes this.

Reply:
Thank you for clarifying. Let's leave as is.